code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -33,7 +33,12 @@
 	layer = MOB_LAYER - 0.1
 	var/removingairlock = 0
 
-
+/mob/living/simple_animal/hostile/zombie/darkholme //BOY♂NEXT♂RUIN
```

Comment:
Could you please add goofs new macho man wrestling belt to the mobs butcher results?

/obj/item/weapon/storage/belt/champion/wrestling

That or just throw one or two on the floor in the gym